.travis.yml

```diff
@@ -2,6 +2,10 @@ language: python
 sudo: false
 python:
     - "2.7"
+env:
+  global:
+    - CF_DEPLOY_USER=fec_deployer
```

Comment:
Follow-up to this is disregard, it's a non-issue given the name is printed in other logs and generally available, and as I noted above with the example, it's caused us issues in the past as it is.